### PR TITLE
build(telemetry): Release 0.4.0

### PR DESCRIPTION
**Pull request by DimedS**
## Description
Release `kedro-telemetry 0.4.0`

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the relevant `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: kedro-telemetry/RELEASE.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Upcoming release 0.4.0
+# Upcoming release
+
+# Release 0.4.0
 * Updated the plugin to generate an unique UUID for each user of `kedro-telemetry`.
```
Comment (noklam) [inline kedro-telemetry/RELEASE.md:4]: I feel like we need to add more docs, we should also update the docs (maybe in README.md too). Do we now have `.telemetry` and `telemetry.toml` at the same time?

Comment (DimedS) [reply kedro-telemetry/RELEASE.md:4]: I agree. It appears we need to revise the Kedro documentation to reflect the updated method for identifying user identity with storing it in the `telemetry.toml` file. I don't think that we should change `README.md` to save consistency.

`Hashed Username: An anonymized representation of the user’s computer username.`

Comment (astrojuanlu) [reply kedro-telemetry/RELEASE.md:4]: xref https://github.com/kedro-org/kedro/pull/3805

**Comment by noklam** [comment]
Anything left to do here?

**Review by ankatiyar — APPROVED** [review]

**Comment by astrojuanlu** [comment]
> Anything left to do here?

Approving reviews!

**Review by ElenaKhaustova — APPROVED** [review]
🚀

**Pull request merged**